File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void){

int height=-1;

    while(height<0 || height>23) {
        printf("hello, write integer up to 23\n");
        height = GetInt();
    }
    for(int line=0; line<height; line++) {
        for(int space=1; space<height-line; space++){
            printf(" ");
        }
        for(int sharp=0; sharp<(height-(height-line)+1); sharp++){
            printf("#");
        }
        printf("#\n");
    }
    return 0;
}

File: pset6/mario.py
height=-1   
while height<0 or height>23:
    print("hello, write integer up to 23 ", end="")
    while True:
        try:
            height = input()
            height = int(height, 10)
            break
        except ValueError:
            pass
        print("Retry: ", end="")
     
for line in range (height):
    print(" "*(height-line)+"#"*line+"##")
    
